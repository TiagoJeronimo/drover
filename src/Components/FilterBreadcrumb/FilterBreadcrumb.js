import React from 'react'
import './FilterBreadcrumb.style.scss'

const FilterBreadcrumb = ({filterCategory, filterName, handleCloseFilter}) => (
    <div className='filterBreadcrumb-container'>
        {console.log("filter", filterName)}
        {filterName}
        <button 
            className='filterBreadcrumb-button'
            onClick={() => handleCloseFilter(filterCategory)}
        >
            X
        </button>
    </div>
)

export default FilterBreadcrumb