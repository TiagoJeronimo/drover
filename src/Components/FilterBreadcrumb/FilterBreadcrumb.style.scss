
.filterBreadcrumb-container {
    background-color: rgb(245, 245, 245);
    padding: 8px 4px;
    margin-right: 8px;
}

.filterBreadcrumb-button {
    background: none;
    border: none;
    color: rgb(0, 30, 255);
    font-size: 16px;
    cursor: pointer;
    text-decoration: none;
}