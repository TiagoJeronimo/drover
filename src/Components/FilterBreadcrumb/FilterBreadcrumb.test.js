import React from 'react';
import { mount } from 'enzyme';

import FilterBreadcrumb from './FilterBreadcrumb';

it('FilterBreadcrumb with normal props', () => {
    const props = {
        filterCategory: 'vehicle_make',
        filterName: 'car',
        handleCloseFilter: jest.fn(),
    };
    const component = (
        <FilterBreadcrumb {...props} />
    );
    const html = mount(component).html();
    expect(html).toMatchSnapshot();
  });

