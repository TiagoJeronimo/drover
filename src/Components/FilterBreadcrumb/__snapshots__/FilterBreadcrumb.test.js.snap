// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterBreadcrumb with normal props 1`] = `"<div class=\\"filterBreadcrumb-container\\">car<button class=\\"filterBreadcrumb-button\\">X</button></div>"`;
